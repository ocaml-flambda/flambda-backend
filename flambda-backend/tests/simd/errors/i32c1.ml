open Stdlib

type t = int32x4

external const1 : int32 -> t = "" "caml_int32x4_const1"
        [@@noalloc] [@@unboxed] [@@builtin]

let _ = const1 (Sys.opaque_identity 1l)
