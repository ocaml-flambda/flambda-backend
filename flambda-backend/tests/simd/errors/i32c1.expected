File "i32c1.ml", line 1:
Error: Did not find constant int arguments for caml_int32x4_const1
